package.json

```diff
@@ -77,5 +80,6 @@
   },
   "dependencies": {
     "dotnet-2.0.0": "^1.4.4"
-  }
-}
+  },
+  "main": "./dist/lib/msRestAzure.js"
```

Comment:
How'd this get here?

Reply:
Ah, must've snuck in during earlier development. I'll remove it now.

Reply:
@RikkiGibson Fixed in https://github.com/Azure/autorest.typescript/pull/89/commits/de2bd90f3971719f18eef040ad0f3bcaa1db6f82.